Error: Installing serverless npx template for CubeJS results in an error

Update: Looks like this happens only on Windows OS.

`npm install --save @cubejs-backend/cubestore-driver`


> @cubejs-backend/cubestore@0.26.81 postinstall '\node_modules\@cubejs-backend\cubestore'
> ./bin/post-install

'.' is not recognized as an internal or external command,
operable program or batch file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @cubejs-backend/cubestore@0.26.81 postinstall: `./bin/post-install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @cubejs-backend/cubestore@0.26.81 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

--- a/rust/package.json
+++ b/rust/package.json
@@ -16,7 +16,7 @@
     "watch": "tsc -w",
     "lint": "eslint js-wrapper/* --ext .ts,.js",
     "lint:fix": "eslint --fix js-wrapper/* --ext .ts,js",
-    "postinstall": "./bin/post-install"
+    "postinstall": "bin/post-install"
   },
   "files": [
     "dist",

